similar/editor/med.cpp

```diff
@@ -878,10 +903,13 @@ static void close_editor()
 			break;
 
 		case 3:
-			set_screen_mode(SCREEN_GAME);		//put up game screen
-			Game_mode = GM_EDITOR;
-			editor_reset_stuff_on_level();
-			N_players = 1;
+			if (!Game_wind)	// if we're already playing a game, don't touch!
```

Comment:
Noted. Later on I want to make and use a proper saved game when loading the editor from within a game, so will change it then.